src/DGtal/geometry/curves/estimation/FunctorsLambdaMST.h

```diff
@@ -0,0 +1,94 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file FunctorsLambdaMST.h
+ * @author Kacper Pluta (\c kacper.pluta@esiee.fr )
+ * Laboratoire d'Informatique Gaspard-Monge - LIGM, A3SI, France
+ *
+ * @date 2014/10/08
+ *
+ * Header file for module FunctorsLambdaMST.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(FunctorsLambdaMST_RECURSES)
+#error Recursive header files inclusion detected in FunctorsLambdaMST.h
+#else // defined(FunctorsLambdaMST_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define FunctorsLambdaMST_RECURSES
+
+#if !defined FunctorsLambdaMST_h
+/** Prevents repeated inclusion of headers. */
+#define FunctorsLambdaMST_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <functional>
+#include "DGtal/base/Common.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+
+/////////////////////////////////////////////////////////////////////////////
+// class FunctorsLambdaMST
+/**
+ * Description of class 'FunctorsLambdaMST' <p>
+ * \brief Aim:
```

Comment:
It is a namespace, not a class. Please add a short description or a reference to some doc.